test/burn/org/apache/cassandra/utils/memory/LongBufferPoolTest.java

```diff
@@ -129,16 +165,22 @@ void testOne() throws Exception
                         return;
                     }
 
-                    BufferPool.put(buffer);
-                    burn.add(buffer);
+                    // 50/50 chance of returning the buffer from the producer thread, or
+                    // pass it on to the consumer.
+                    if (rand.nextBoolean()) {
```

Comment:
According to the code style, you can skip braces here. We also follow follow Sun Style so braces go on new lines.
Reference: http://cassandra.apache.org/doc/latest/development/code_style.html

Reply:
fixed